### PR TITLE
Fix distlib-css-bundle

### PR DESCRIPTION
**Pull request by ger-benjamin**
For GSNGEO-21
Makes more sens.

[Examples](https://camptocamp.github.io/ngeo/refs/pull/9506/merge/examples/)
[Storybook](https://camptocamp.github.io/ngeo/refs/pull/9506/merge/storybook/)
[API help](https://camptocamp.github.io/ngeo/refs/pull/9506/merge/api/apihelp/apihelp.html)
[API documentation](https://camptocamp.github.io/ngeo/refs/pull/9506/merge/apidoc/)

### CODE REVIEW

**Review by sbrunner — CHANGES_REQUESTED** [review]
His don't remove the hash!

**Comment by ger-benjamin** [comment]
Yes, ok, I forgot why you did that.
But cp dist/desktop-*.css distlib/src/controllers/desktop.css doesn't work => destination is not a folder

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "distlib": "tsc --pretty",
     "distlib-images": "copyfiles --verbose src/**/*.svg src/*.svg src/**/*.png src/*.png distlib",
     "distlib-css": "copyfiles --verbose src/**/*.css src/*.css src/**/*.scss src/*.scss distlib",
-    "distlib-css-bundle": "cp dist/desktop-*.css distlib/src/controllers/desktop.css && cp dist/mobile-*.css distlib/src/controllers/mobile.css && cp dist/iframe_api-*.css distlib/src/controllers/iframe_api.css",
+    "distlib-css-bundle": "cp dist/desktop-*.css distlib/src/controllers/ && cp dist/mobile-*.css distlib/src/controllers/ && cp dist/iframe_api-*.css distlib/src/controllers/",
```
Comment (sbrunner) [inline package.json:21]: Then:
```suggestion
    "distlib-css-bundle": "mkdir -p distlib/src/controllers/ && cp dist/desktop-*.css distlib/src/controllers/desktop.css && cp dist/mobile-*.css distlib/src/controllers/mobile.css && cp dist/iframe_api-*.css distlib/src/controllers/iframe_api.css",
```

Comment (ger-benjamin) [reply package.json:21]: I don't understand...
With `file1.css`, `file2.css`, you can't do `cp file*.css final.css`
Or did I miss something ? You copy multiple file into one ?

Or maybe do you want `cat dist/desktop-*.css > distlib/src/controllers/desktop.css` ?

Comment (sbrunner) [reply package.json:21]: You shouldn't get two files, If you get it, we can add en rm in dist target.

Comment (ger-benjamin) [reply package.json:21]: Ahhhh ok, now I understand. My `dist` folder was polluted by several build.
I tought with the `*.css` you wanted to select multiple files, but no, you did that to select one file with unknown hash.
That explain why the CI passe.
That explain all, I close this PR, thanks.

**Comment by ger-benjamin** [comment]
I was confused because multiple file in my dist, instead of one.
this PR is useless and based on a missunderstandnis

**Pull request closed without merging**